components/common/btn/followButtons.tsx

```diff
@@ -1,30 +1,28 @@
 import styled from '@emotion/styled';
-
-interface FollowBtnProps {
-  bgColor: string;
+interface InFollowButtonpropsTypes {
+  backgroundColor: string;
   children: React.ReactNode;
 }
 
-const Follow = styled.button<FollowBtnProps>`
+export default function FollowButtons({ children, backgroundColor }: InFollowButtonpropsTypes) {
+  return <Follow backgroundColor={backgroundColor}>{children}</Follow>;
+}
+
+const Follow = styled.button<InFollowButtonpropsTypes>`
   display: flex;
   align-items: center;
   justify-content: center;
 
   height: 2.5rem;
   padding: 0.7rem 0.9rem;
 
```

Comment:
```suggestion
export default function FollowButtons({ children, isFollow }: InFollowButtonpropsTypes) {
  return <Follow isFollow={isFollow}>{children}</Follow>;
}

const Follow = styled.button<InFollowButtonpropsTypes>`
  display: flex;
  align-items: center;
  justify-content: center;

  height: 2.5rem;
  padding: 0.7rem 0.9rem;
  background-color : ${({isFollow, theme}) => isFollow ? theme.color.red : theme.color.green

```
이런 식은 어때용?
그러면 안에 내용도 상수화시켜서 넣을 수 있을 것 같고,,,